src/bookmarks/background/addition.js

```diff
@@ -50,9 +51,20 @@ export async function createNewPageForBookmark(id, bookmarkInfo) {
     }
 }
 
-export async function createBookmarkByExtension(url) {
+export async function createBookmarkByExtension(url, tabId) {
```

Comment:
This is the same function used for the bookmark event from overview right? If so we'll need to switch logic so it works both with `tabId` and without (or just make a separate `createBookmarkByTab` or something)